openmls/src/framing/validation.rs

```diff
@@ -315,8 +315,38 @@ impl VerifiedExternalMessage {
 
 /// Message that contains messages that are syntactically and semantically correct.
 /// [StagedCommit] and [StagedProposal] can be inspected for authorization purposes.
+#[derive(Debug)]
 pub enum ProcessedMessage {
-    ApplicationMessage(Vec<u8>),
+    ApplicationMessage(ApplicationMessage),
     ProposalMessage(Box<StagedProposal>),
     StagedCommitMessage(Box<StagedCommit>),
 }
+
+/// Application message received through a [ProcessedMessage].
```

Comment:
According to [this](https://doc.rust-lang.org/rustdoc/linking-to-items-by-name.html), backticks will be stripped anyway which is why I stopped adding them. This will obviously need a pass to harmonise things in the end.

Reply:
Ah that got stabilised. Never mind me.